### ISSUE TITLE
animation json files

### ISSUE THREAD

**Issue by ghost** [grumpygamer/DeloresDev#55]
Hi

Is there a tool to make/edit the animation json files?

**Comment by grumpygamer** [comment]
Yes, there is a tool but it's so bad it's barely usable.  We mostly edited the files by hand.
